spectrum_io/search_result/search_results.py

```diff
@@ -69,18 +69,19 @@ def generate_internal(self, tmt_labeled: str, out_path: Optional[Union[str, Path
         """
         if out_path is None:
             # convert and return
-            return self.read_result(tmt_labeled)
+            return self.read_result(tmt_labeled, custom_stat_mods, custom_var_mods)
```

Comment:
here, sth. is wrong, because custom_stat_mods and custom_var_mods are both the dictionaries that contain a mapping from the search_engine specific mod_name to a tuple of internal name and mass, but you require only the mapping from search_engine to internal name here! So this is where you need to have some mapping function that forwards only what you need. Compare with the type annotation for the custom read_results functions, they expect Dict[str, str], not Dict[str, Tuple[str, float]]!